### ISSUE TITLE
"Open" dialogs are somehow treated as noWindowActive

### ISSUE THREAD

**Issue by kupiqu** [kotelnik/plasma-applet-active-window-control#53]
This makes the applet to show plasmoid.configuration.noWindowText ("Plasma Desktop"). I think it would be more informative to show the title of the active "open" dialog instead.

**Comment by kotelnik** [comment]
You are right, I know about this problem but didn't have time yet to resolve it. I'll look into it.
